hack/update-json-library.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# This script can be called via ./hack/update-json-library.sh` to update the
+# go-json-experiment fork in the repo. 
+
+# HASH should be updated to the desired commit ID in https://github.com/go-json-experiment/json.
+HASH="540f014424240312547dcccddf11a8229ca0f463"
+
+GO_JSON_EXPERIMENT_DIR="pkg/internal/third_party/go-json-experiment/json"
+rm -rf $GO_JSON_EXPERIMENT_DIR
+git clone --depth 1 https://github.com/go-json-experiment/json $GO_JSON_EXPERIMENT_DIR
```

Comment:
I think `--depth 1` likely won't work with a hash other than HEAD

Reply:
Thanks for the catch, updated